### PR TITLE
Distribute vulture as a wheel

### PR DESCRIPTION
**Pull request by RJ722**
Closes: #78

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation in the README file.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by jendrikseipp** [comment]
I think it would be good to add tests for the wheel files on travis-ci and appveyor. I think the tests should:

* Build the wheel file
* Install the local wheel file with pip
* Run the installed vulture version on the local vulture code.

File: .travis.yml [hunk 1]
```diff
@@ -15,7 +15,11 @@ matrix:
   - python: 3.7-dev
 install:
   - pip install tox
+  - python setup.py bdist_wheel
+  - pip install --use-wheel dist/*.whl
```
Comment (jendrikseipp) [inline .travis.yml]: can we use `pip install --use-wheel .` ?

File: .travis.yml [hunk 2]
```diff
@@ -15,7 +15,11 @@ matrix:
...
 script:
+  # Run the installed version of vulture on local code.
+  - vulture vulture
```
Comment (jendrikseipp) [inline .travis.yml]: can we include the tests here?

Comment (RJ722) [inline .travis.yml]: Which ones?

File: .travis.yml (hunk 1 above)
Comment (RJ722) [inline .travis.yml]: Oh! Yes, Thanks! Done making changes! :-)

File: .travis.yml [hunk 3]
```diff
@@ -15,7 +15,11 @@ matrix:
   - python: 3.7-dev
 install:
   - pip install tox
+  - python setup.py bdist_wheel
+  - pip install --use-wheel .
 script:
+  # Run the installed version of vulture on local code.
+  - vulture vulture
```
Comment (jendrikseipp) [inline .travis.yml]: I mean `vulture vulture/ tests/`.

Comment (RJ722) [inline .travis.yml]: Oh! I misunderstood it as "can we run the tests here?" 😅 

Done!

**Comment by RJ722** [comment]
The windows build is failing and I can't even SSH into it to experiment. :-(

**Comment by jendrikseipp** [comment]
Code looks good. Can you try to fix the build on appveyor, please?

**Comment by RJ722** [comment]
Hi @jendrikseipp, I was just reading on "How to build python wheels on windows?". I'll update the PR if something comes up.

**Comment by jendrikseipp** [comment]
We probably need to install a newer version of pip or setuptools on appveyor.

    pip install -U pip setuptools

**Comment by RJ722** [comment]
> We probably need to install a newer version of pip or setuptools on appveyor.
```
pip install -U pip setuptools
```

Oh! Let me give it a try. Thanks!

Also, I had a little chat with @jayvdb, he told that using `pip install --use-wheel` doesn't use the wheel generated in the prior setup. We need to be more explicit.  (For example: https://github.com/coala/coala/blob/master/.travis.yml#L71)

Should I change it back to `./dist/*.whl`?

**Comment by jendrikseipp** [comment]
Sounds good. Probably best to use `./dist/vulture-*.whl`.

**Comment by RJ722** [comment]
Upgrading setuptools is not working. :-(

**Comment by RJ722** [comment]
Hmm, we needed install the `wheel` before running `python setup.py bdist_wheel`. :-)

But now, how do we specify the glob pattern in windows? :-(
@jendrikseipp maybe you have an idea?

**Comment by jendrikseipp** [comment]
Maybe this SO answer helps? https://stackoverflow.com/a/27885547

**Comment by RJ722** [comment]
Working! :-)

Thanks!

**Comment by jendrikseipp** [comment]
Awesome, thanks!

**Pull request merged**